wagtail/users/views/users.py

```diff
@@ -90,9 +92,32 @@ def index(request, *args):
 
     if 'ordering' in request.GET:
         ordering = request.GET['ordering']
-
-        if ordering == 'username':
+        if ordering == "-name":
+            users = users.order_by('-last_name', '-first_name')
+        elif ordering == 'username':
             users = users.order_by(User.USERNAME_FIELD)
+        elif ordering == '-username':
+            users = users.order_by('-'+User.USERNAME_FIELD)
+        elif ordering == 'level':
+            superusers = users.filter(is_superuser=True)
+            staff = users.filter(is_staff=True).exclude(id__in=superusers)
```

Comment:
Why not `filter(is_staff=True, is_superuser=False)`?

Reply:
oo yes that would be better, I'll make those changes.

Reply:
with the natural order thing - taking the is_active ascending as an example, do you mean how all the active users are sorted against each other? If so I think I get you, I'll add that but let me know if you meant something else.

Reply:
I've added a secondary sort to all the columns and I do agree it's a big improvement. I've updated the main comment to reflex the changes. I'm worried the code is a little verbose as Django clears the order when you execute a new order_by. Not sure if there's a more elegant way than the way I've done it.